vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -69,11 +112,26 @@ func CreateProcessInComputeSystem(id string, params CreateProcessParams) (proces
 
 	logrus.Debugf(title+" - Calling the procedure itself %s %s", id, paramsJson)
 
+	var stdinHandle, stdoutHandle, stderrHandle syscall.Handle
+	var stdinParam, stdoutParam, stderrParam uintptr
+	if useStdin {
+		stdinParam = uintptr(unsafe.Pointer(&stdinHandle))
```

Comment:
No. We are receiving a handle as an out parameter from the native C routine. So we need to pass a *syscall.Handle (which must be cast to uintptr to match the signature of .Call).